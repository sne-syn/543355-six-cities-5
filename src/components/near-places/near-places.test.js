import React from 'react';
import renderer from 'react-test-renderer';
import NearPlaces from './near-places';

jest.mock(`../list/list`, () => `List`);

test(`Render NearPlaces`, () => {
  const tree = renderer
    .create(
        <NearPlaces
          type={`near-places`}
          offers={[]}
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
