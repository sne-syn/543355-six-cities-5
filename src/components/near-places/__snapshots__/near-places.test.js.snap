// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render NearPlaces 1`] = `
<section
  className="near-places places"
>
  <h2
    className="near-places__title"
  >
    Other places in the neighbourhood
  </h2>
  <List
    offers={Array []}
    type="near-places"
  />
</section>
`;
